cns-hook

```diff
@@ -125,6 +139,13 @@ function deploy_cert {
 		zlogin "${vmuuid}" svcadm restart adminui
 		echo "OK: adminui certificate deployed, and adminui restarted" >&2
 		;;
+	sdcsso)
+		vmuuid="$(vmadm lookup alias=sdcsso)"
+                cp "${certdir}/fullchain.pem" "/zones/${vmuuid}/root/opt/smartdc/sdcsso/ssl/cert.pem"
+                cp "${certdir}/privkey.pem" "/zones/${vmuuid}/root/opt/smartdc/sdcsso/ssl/key.pem"
+		zlogin "${vmuuid}" svcadm restart sdcsso
+		echo "OK: sdcsso certificate deployed, and sdcsso restarted" >&2
```

Comment:
It looks like these files are not kept between reprovisions of the sdcsso zone, is that correct? We might want to consider making a branch that injects these as server-side SAPI manifests then, as well, like we do for CloudAPI. That can be a separate thing though.